docker-compose/bind9/config/yourdomain.zone

```diff
@@ -0,0 +1,15 @@
+$TTL   300  ; 5 minutes
```

Comment:
Having such a short default TTL on all records of the zone implies a certain operational risk. Also, when 5 minutes is meant, why not just write `5m` and drop the redundant comment.